packages/jupyter/src/plotly.ts

```diff
@@ -30,9 +30,9 @@ export function isPlotly(outputs: IOutput[]) {
 }
 
 export function usePlotlyPassively(rendermime: IRenderMimeRegistry, outputs: IOutput[]) {
-  const [loaded, setLoaded] = useState(false);
-
   const isPlotlyOutput = isPlotly(outputs);
```

Comment:
fixed additional issue found during testing, also introduced by the recent plotly change